great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -1221,3 +1199,23 @@ def list_profilers(
         if ge_cloud_mode:
             return profiler_store.list_keys()
         return [x.configuration_key for x in profiler_store.list_keys()]
+
+
+def _validate_builder_override_config(builder_config: dict):
+    """
+    In order to insure successful instantiation of custom builder classes using "instantiate_class_from_config()",
+    candidate builder override configurations are required to supply both "class_name" and "module_name" attributes.
+
+    :param builder_config: candidate builder override configuration
+    :raises: ProfilerConfigurationError
+    """
+    if not all(
+        [
+            isinstance(builder_config, dict),
+            "class_name" in builder_config,
+            "module_name" in builder_config,
+        ]
```

Comment:
Nitpick - can we make this a generator by using parens instead?

Reply:
@cdkini Committed (among a few other conversions from list to generators) -- thanks!

Reply:
Big 🙇 to @cdkini with huge thanks for thoughtful and balanced reviews.